docs/source/overview/environments.rst

```diff
@@ -57,6 +57,8 @@ Classic environments that are based on IsaacGymEnvs implementation of MuJoCo-sty
     |                  |                             | and perceptive inputs                                                   |
     |                  | |cartpole-depth-link|       |                                                                         |
     |                  |                             |                                                                         |
+    |                  | |cartpole-resnet-link|      |                                                                         |
```

Comment:
Maybe it is even worth adding this into a separate row, where you add a description that it uses a resnet for feature extraction?

Reply:
Yup! On it! ;)